src/error.rs

```diff
@@ -1,57 +1,58 @@
-use errno::errno;
-use std::{
-    ffi::{IntoStringError, NulError},
-    ptr,
-    string::FromUtf8Error,
-};
+// Copyright 2017 Red Hat, Inc.
 
-#[derive(thiserror::Error, Debug)]
-pub enum BlkidError {
-    #[error("blkid error, value is '{val}'")]
-    LibBlkid { val: i32 },
+// Licensed under the MIT license <LICENSE or
+// http://opensource.org/licenses/MIT> This file may not be copied, modified,
+// or distributed except according to those terms.
```

Comment:
Is this necessary? The crate already has a `LICENSE` file included.

Reply:
I agree, we can remove this in all files at all.

Reply:
Fixed.